patches/minecraft/net/minecraft/world/chunk/Chunk.java.patch

```diff
@@ -185,15 +185,23 @@
      }
  
      public void func_76623_d()
-@@ -871,6 +873,7 @@
+     {
++        // FORGE - Fix for MC-92916
++        final List<net.minecraft.entity.player.EntityPlayer> players = new java.util.ArrayList<net.minecraft.entity.player.EntityPlayer>();
++        java.util.Arrays.stream(field_76645_j).forEach(multimap -> multimap.func_180215_b(net.minecraft.entity.player.EntityPlayer.class).forEach(players::add));
```

Comment:
Could you use `.collect(Collectors.toList())` to construct the `players` List?
That's getting nitpicky though, I think this patch is pretty good. If you have time to clean it up more that's great.

Reply:
I'll play with it, but I'm not sure I can.

Reply:
Just do `final List<EntityPlayer> players = Lists.newArrayList(multimap.func_180215_b(net.minecraft.entity.player.EntityPlayer.class));`

Reply:
I had a play with this and got it down to one line with         java.util.Arrays.stream(entityLists).flatMap(multimap -> java.util.stream.StreamSupport.stream(multimap.getByClass(net.minecraft.entity.player.EntityPlayer.class).spliterator(), false)).forEach(player -> world.updateEntityWithOptionalForce(player, false));
Sorry for formatting

Reply:
I had a play with this and got it down to one line with java.util.Arrays.stream(entityLists).flatMap(multimap -> java.util.stream.StreamSupport.stream(multimap.getByClass(net.minecraft.entity.player.EntityPlayer.class).spliterator(), false)).forEach(player -> world.updateEntityWithOptionalForce(player, false));
Sorry for formatting

Reply:
@liach That doesn't work because there are two lists to loop through. That line assumes you already have a reference to the internal list. We are working with a ```List<List<Entity>>```

Reply:
why would you want to condense this piece of code into a one-liner that takes any coder 10 minutes to decode in their head and compiles into something that is slower than boring old-fashioned loops?

Reply:
I think we may use `Iterables.concat`